provider/maas/Makefile

```diff
@@ -8,10 +8,7 @@ bridgescript.go: add-juju-bridge.py Makefile
 	$(RM) $@
 	echo -n '// This file is auto generated. Edits will be lost.\n\n' >> $@
 	echo -n 'package maas\n\n' >> $@
-	echo -n '//go:generate make -q\n\n' >> $@
```

Comment:
The go generate mechanism never materialised in the build because, for a long time, we were stuck on versions of go that did not support go generate.